README.md

```diff
@@ -90,46 +91,46 @@ class CreateCuriosities < ActiveRecord::Migration
 
-Les deux dernières colonnes (created_at et updated_at) sont créées grace à la ligne ````t.timestamps null: false````, ajoutée automatiquement par Ruby On Rails. 
+On peut créer les deux dernières colonnes (````created_at```` et ````updated_at````) en un coup grâce à un raccourci : la ligne ````t.timestamps null: false````. Il est intéressant de noter que vous n'avez pas besoin de "penser" à ces deux champs : rien qu'avec leur nom, Ruby on Rails détecte que ce sont des champs pour garder les dates de création et modification (si vous aviez des champs de création / modification qui ne sont PAS nommés ainsi, il faudrait le faire soi-même).
 
-Lorsque des migrations ont été effectuées sur la base de données de votre application, il est possible de retrouver sa structure complète dans le fichier /db/schema.rb
+Lorsque des migrations ont été effectuées sur la base de données de votre application, il est possible de retrouver sa structure complète dans le fichier ````/db/schema.rb````.
 
-Ouvrez ce fichier. Vous pouvez constater que vous retrouvez la méthode de créationd e la table ````Curiosities````, avec tout ce qui la compose. 
+Ouvrez ce fichier. Vous pouvez constater que vous retrouvez la méthode de création de la table ````Curiosities````, avec tout ce qui la compose. 
 
-Astuce : 
-Une migration a un identifiant. Cela permet à votre application de savoir quelle est la derniere migration qui a été processée, pour éviter de la rejouer. Cet identifiant de migration est visible dans le nom du fichier de la migration. 
+> Astuce : 
+> Une migration a un identifiant. Cela permet à votre application de savoir quelle est la dernière migration qui a été traitée, pour éviter de la rejouer. Cet identifiant de migration est visible dans le nom du fichier de la migration. 
 
-Vous pouvez voir à quelle migration vous en etes dans le fichier /db/schema.rb avec ````version: 20160618174815````.
+Vous pouvez voir à quelle migration vous en êtes dans le fichier ````/db/schema.rb```` avec ````version: 20160618174815````.
 
-#### Active Record et les modèles
+#### ActiveRecord et les modèles
 
-Active Record est la partie de Ruby On Rails qui permet de manipuler les informations stockées en base de données avec des modèles. 
+ActiveRecord est la partie de Ruby On Rails qui permet de manipuler les informations stockées en base de données avec des classes Ruby (qu'on appelle les modèles).
 
-Cela permet de représenter les attributs d'une table de la base de données dans un modèle, pour avoir accès à chacune des lignes de cette table et pouvoir les manipuler. 
+Cela permet de représenter les attributs d'une table de la base de données dans un modèle, pour avoir accès à chacune des lignes de cette table (comme une ligne dans Excel !) et pouvoir les manipuler. 
 
-Astuce : Avec Ruby On Rails, une classe s'appelle aussi un modèle. 
+> Astuce : Avec Ruby On Rails, ces classes s'appellent aussi des modèles. 
```

Comment:
Well, this line is useless now as we've :
ActiveRecord est la partie de Ruby On Rails qui permet de manipuler les informations stockées en base de données avec des classes Ruby (qu'on appelle les modèles).